.env.local

```diff
@@ -5,7 +5,7 @@ API_URL_PREFIX='http://127.0.0.1:3000'
 # pagoPA RESTful API
 PAGOPA_API_URL_PREFIX='http://127.0.0.1:3000/wallet'
 # pagoPA test/env RESTful API
-PAGOPA_API_URL_PREFIX_TEST='https://acardste.vaservices.eu:443/pp-restapi-CD'
+PAGOPA_API_URL_PREFIX_TEST='https://uat.wisp2.pagopa.gov.it/pp-restapi-CD'
```

Comment:
Why we are using 12.0.0.1 for production and real URL for test?

Reply:
on the local environment we don't have any differences between we may repeat `http://127.0.0.1:3000/wallet` btw nothing would change